src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# ListBoxOption:
+
+Provides an ListBoxOption component that is passed as a child to the ListBox component.
+
+## API
```

Comment:
My one concern is that focusing on the ListBox is the only way to interact with it with the keyboard. The pattern is that the ListBox gets focus and then you navigate through the ListBoxOptions with the arrow keys. The ListBox is also supposed to auto-select the first ListBoxOption if you focus on the ListBox for the first time, and I was a little worried about an onFocus prop providing the opportunity to break accessibility without the consumer knowing (e.g. they choose to select the last option onFocus with the selectedOptionIndex prop or something). I definitely see the use for onFocus and onBlur, and they seem to be common props to expore. Do you think we should just expose them and put the onus on the user to know when they're breaking accessibility patterns?

Reply:
Definitely something to be concerned about 👍 .

I guess my opinion is that if there's a feature where somebody **could** misuse the API I'm not sure that is reason enough to not support such a feature. Particularly if it's

1. Well-documented: So maybe in the documentation for `onFocus/Blur` call out that certain `state` changes should not occur in certain circumstances.
2. Supported by the API itself: Maybe there are console warnings or something in the cases where accessibility patterns are broken - this could be too complex though.

All that being said, I also think that if there's not an initial use case, it might make sense to punt - it's (almost) always easier to add API methods vs. removing them.